### PR TITLE
Upgrade from v0.9.37 to  v0.9.40

### PR DESCRIPTION
**Pull request by valentinfernandez1**

### CODE REVIEW

**Comment by bernardoaraujor** [comment]
can you please make sure the tests are passing?

File: src/lib.rs [hunk 1]
```diff
@@ -193,7 +193,6 @@ pub mod pallet {
     #[pallet::pallet]
     #[pallet::without_storage_info]
     // https://stackoverflow.com/questions/70206199/substrate-tutorials-trait-maxencodedlen-is-not-implemented-for-vecu8
```
Comment (bernardoaraujor) [inline src/lib.rs:195]:
```suggestion
```

**Comment by bernardoaraujor** [comment]
close #25

**Comment by hbulgarini** [comment]
@valentinfernandez1 please also create the branch `polkadot-v0.9.40` so we can import the specific branch from the runtime.

**Comment by valentinfernandez1** [comment]
I'm not part of the repo members

**Comment by bernardoaraujor** [comment]
> @valentinfernandez1 please also create the branch `polkadot-v0.9.40` so we can import the specific branch from the runtime.

done

**Pull request merged**